creator-cards/brian-mckeiver.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Brian McKeiver",
+    "sectionTitle": null,
+    "countryCode": "us",
+    "slogan": "",
+    "type": "CREATOR",
+    "bio": "Brian McKeiver is Co-Owner, Solutions Architect, & Microsoft Azure MVP  at BizStream, a digital agency in Allendale, MI.",
+    "tags": ".NET, Azure, CMS",
+    "socials": {
+      "youtube": "https://www.youtube.com/c/kontentrocks",
+      "linkedin": "https://www.linkedin.com/in/bmckeiver/",
+      "twitter": "https://twitter.com/mcbeev",
+      "github": "https://github.com/mcbeev",
+      "mastodon": "",
+      "sessionize": "https://sessionize.com/bmckeiver"
+    },
+    "channels": [
+      {
+        "name": "Blog",
+        "url": "https://www.mcbeev.com/"
+      }
+    ],
+    "feeds": [{
+        "type": "LinkTagRssType",
+        "url": "https://www.mcbeev.com/api/rss/articles"  
```

Comment:
Can you make sure to give us a feed that only contains your .NET and related content please?